Gemfile

```diff
@@ -0,0 +1,6 @@
+source "https://rubygems.org"
```

Comment:
Please get the latest version from the `template-formula`, which is `rubocop` certified!